scripts/telnet-fingerprint.nse

```diff
@@ -0,0 +1,96 @@
+local stdnse = require "stdnse"
+local shortport = require "shortport"
+local comm = require "comm"
+local nmap = require "nmap"
+
+description = [[
+Attempts to fingerprint an open telnet service based on Telnet commands sent
+by the server upon connection. See RFC854 for more details.
+
+Original idea from telnetfp by Palmers of Team TESO.
+
+Please send new or incorrect fingerprint data to daniel@planethacker.net
+]]
+
+---
+-- @usage
+-- nmap --script telnet-fingerprint <target>
+
+-- @output
+-- 23/tcp open  telnet
+-- | telnet-fingerprint:
+-- |   Fingerprint: 255 252 1
+-- |_  Match: HP JetDirect
+
+author = "Daniel Roberson"
+license = "Same as Nmap--See https://nmap.org/book/man-legal.html"
+categories = {"discovery", "safe"}
+
+
+-- Fingerprint table. Please keep in alphabetical order!
+fp_table = {}
+fp_table["255 251 1"] = "APC, Windows CE"
```

Comment:
Add reference links for these fingerprints in the description above.

Reply:
This is the only item I was unsure about. I do not know what you mean by adding reference links in the description.